lib/datalifecycle/purge/workflow.go

```diff
@@ -12,17 +12,17 @@ import (
 
 // ConfigureManager registers the purge workflow schedule and job with purge
```

Comment:
We should fix the comment here. We're registering a workflow and task, not a schedule.